spmd/tensor/ops/dropout.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+import torch
+from typing import Tuple
+from spmd.tensor.ops.utils import register_impl
+from spmd.tensor.api import Tensor
+
+
+@register_impl("aten.native_dropout.default")
+def _dist_dropout(self: Tensor, p: float, train: bool) -> Tuple[Tensor, Tensor]:
+    print(self.placements)
+    self_placement = self.placements[0]
+    if self_placement.is_partial():
```

Comment:
let's error out on both partial and replicated for now, and maybe add an TODO here to explain the CPU/GPU difference and fix later

Reply:
We can still keep this logic since we need this logic for GPU right?

Reply:
Yep, in GPU it will trigger this function